### PR TITLE
Adding 6.0 ServiceFabric specification

### PR DESCRIPTION
**Pull request by samedder**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by anuchandy** [comment]
@samedder can you update the [readme](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/servicefabric/data-plane/readme.md) file to have a tag for this version of swagger and include the updated readme in this PR?

File: specification/servicefabric/data-plane/Microsoft.ServiceFabric/6.0/servicefabric.json [hunk 1]
```diff
@@ -0,0 +1,11892 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Service Fabric Client APIs",
+    "description": "Service Fabric REST Client APIs allows management of Service Fabric clusters, applications and services.",
+    "version": "6.0.*"
+  },
+  "host": "localhost:19080",
```
Comment (anuchandy) [inline specification/servicefabric/data-plane/Microsoft.ServiceFabric/6.0/servicefabric.json]: hot is hardcoded to `localhost:19080`, does this mean the generated client can be used only from within the cluster?

**Comment by anuchandy** [comment]
@samedder any update?

File: specification/servicefabric/data-plane/Microsoft.ServiceFabric/6.0/servicefabric.json (hunk 1 above)
Comment (samedder) [reply specification/servicefabric/data-plane/Microsoft.ServiceFabric/6.0/servicefabric.json]: No I thought that's only when testing the Swagger spec. The generated client accepts any host.

File: specification/servicefabric/data-plane/Microsoft.ServiceFabric/6.0/servicefabric.json [hunk 2]
```diff
@@ -0,0 +1,11892 @@
...
+    "http",
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/$/GetClusterManifest": {
+      "get": {
+        "operationId": "GetClusterManifest",
```
[the remark this answers is not in the document]
Comment (samedder) [reply specification/servicefabric/data-plane/Microsoft.ServiceFabric/6.0/servicefabric.json]: This isn't something we plan to change for the time being.

**Comment by samedder** [comment]
Sorry for the delay, @anuchandy, we're going to send a new PR with some changes that were missed in this PR.

**Pull request closed without merging**